.eslintrc.js

```diff
@@ -1,14 +1,6 @@
 module.exports = {
   extends: ['@addepar', '@addepar/eslint-config/ember'],
   parser: 'babel-eslint',
-  parserOptions: {
-    ecmaFeatures: {
-      legacyDecorators: true,
-    },
-  },
-  env: {
-    es6: true,
-  },
```

Comment:
babel-eslint's docs note that the ESLint official parser's `parserOptions` aren't necessarily supported when using this alternative parser. I found that these options didn't seem to be needed, so I removed them.
Docs: https://github.com/babel/babel-eslint#setup